code/modules/mob/living/carbon/human/species_types/shadowpeople.dm

```diff
@@ -70,15 +69,17 @@
 
 /obj/item/light_eater
 	name = "light eater"
-	icon_state = "arm_blade"
-	item_state = "arm_blade"
+	desc = "A mass of tentacles, cloaked in thick, black smoke. Attacking a target with it will extinguish any lights they're carrying."
+	icon_state = "tentacle"
+	item_state = "light_eater"
 	force = 25
 	armour_penetration = 35
 	lefthand_file = 'icons/mob/inhands/antag/changeling_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/antag/changeling_righthand.dmi'
 	flags_1 = ABSTRACT_1 | NODROP_1 | DROPDEL_1
 	w_class = WEIGHT_CLASS_HUGE
 	sharpness = IS_SHARP
+	attack_verb = list("consumed", "devoured", "eaten")

```

Comment:
these don't actually make sense as attack verbs for something that isn't an instakill

Reply:
Eaten is the only one that will make no sense grammatically. Everything else can be chalked to eating/devouring only parts of the target.

Reply:
It's named "light eater" so my headcanon is that (with the tentacly look) it bites off chunks of the target, yeah; imagine a squid with a toothy mouth on the inside of the tentacles

Reply:
It turns everything into dust though

Reply:
Spooky eldritch magic